main.c

```diff
@@ -721,6 +727,12 @@ static void print_formatted_result(FILE *stream, const struct slurp_box *result,
 			case 'y':
 				fprintf(stream, "%d", result->y);
 				continue;
+			case 'X':
```

Comment:
Could you add a description of these options to the man page (edit slurp.1.scd) in the FORMAT section?